Docker/README.md

```diff
@@ -0,0 +1,70 @@
+
+# Integration rextporter and skycoin using Docker
+
+## Dockerfile for build rextporter image
```

Comment:
@mauricio1802 
I'm not that is required  put this content here, if you have any particular comment put it in the source file itself, here write doc for end users only, like how to define the listen port or any other thing.

Add a link to this readme from the parent project README